packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1988,6 +1988,10 @@ class SemanticsNode with DiagnosticableTreeMixin {
   /// See also:
   ///
   ///  * [elevation], the actual elevation of this [SemanticsNode].
+  @Deprecated(
+    'This was caches for internal calculation and is no longer needed. '
+    'This feature was deprecated after v3.27.0-0.0.pre.',
+  )
```

Comment:
FWIW, I think we should deprecate/remove the entire elevation/thickness feature from semantics. It's unused and has never been used.